src/display/editor/alt_text.js

```diff
@@ -218,10 +232,12 @@ class AltText {
       // If we've a guessed text and the alt text has never been set, we get a
       // "to-review" been set.
       // Otherwise, we get a "missing".
-      const type =
-        (this.#altText && "added") ||
-        (this.#altText === null && this.guessedText && "to-review") ||
-        "missing";
+      const label = this.#label;
+      const type = label === "review" ? "to-review" : label;
+      this.#editor._reportTelemetry({
+        action: "pdfjs.image.alt_text.image_status_label_displayed",
+        label,
+      });
       button.classList.toggle("done", !!this.#altText);
       AltText._l10nPromise
         .get(`pdfjs-editor-new-alt-text-${type}-button-label`)
```

Comment:
Could we perhaps change the name of the l10n-key, to allow using the `label` directly here instead?

Reply:
I'll do that later, I'll uplift the patch in beta and I'd prefer not touching the localization.
Let me add a TODO.